[ILM] TimeSeriesLifecycleActionsIT.testRolloverAlreadyExists failure

Console log: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-unix-compatibility/os=oraclelinux-7&&immutable/390/console

Build scan: https://gradle-enterprise.elastic.co/s/2knxuomyu6d62

Failure:

```
ava.lang.AssertionError: 

Expected: <{"phase":"hot","action":"rollover","name":"ERROR"}>
     but: was <{"phase":"hot","action":"rollover","name":"check-rollover-ready"}>
Close stacktrace
at __randomizedtesting.SeedInfo.seed([2359606ACADA06B8:3C3BEB9E601FCD94]:0)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT.lambda$testRolloverAlreadyExists$17(TimeSeriesLifecycleActionsIT.java:318)
at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:874)
at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:847)
at org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT.testRolloverAlreadyExists(TimeSeriesLifecycleActionsIT.java:314)
```

Reproduce with:

```
./gradlew ':x-pack:plugin:ilm:qa:multi-node:integTestRunner' --tests "org.elasticsearch.xpack.ilm.TimeSeriesLifecycleActionsIT.testRolloverAlreadyExists" -Dtests.seed=2359606ACADA06B8 -Dtests.security.manager=true -Dtests.locale=sr -Dtests.timezone=America/Atikokan -Dcompiler.java=12 -Druntime.java=11
```

I could not reproduce locally.

--- a/x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java
+++ b/x-pack/plugin/ilm/qa/multi-node/src/test/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java
@@ -26,7 +26,6 @@ import org.elasticsearch.index.IndexSettings;
 import org.elasticsearch.test.rest.ESRestTestCase;
 import org.elasticsearch.xpack.core.ilm.AllocateAction;
 import org.elasticsearch.xpack.core.ilm.DeleteAction;
-import org.elasticsearch.xpack.core.ilm.ErrorStep;
 import org.elasticsearch.xpack.core.ilm.ForceMergeAction;
 import org.elasticsearch.xpack.core.ilm.FreezeAction;
 import org.elasticsearch.xpack.core.ilm.LifecycleAction;
@@ -40,7 +39,6 @@ import org.elasticsearch.xpack.core.ilm.ShrinkAction;
 import org.elasticsearch.xpack.core.ilm.ShrinkStep;
 import org.elasticsearch.xpack.core.ilm.Step.StepKey;
 import org.elasticsearch.xpack.core.ilm.TerminalPolicyStep;
-import org.elasticsearch.xpack.core.ilm.WaitForRolloverReadyStep;
 import org.hamcrest.Matchers;
 import org.junit.Before;
 
@@ -288,40 +286,6 @@ public class TimeSeriesLifecycleActionsIT extends ESRestTestCase {
         assertBusy(() -> assertTrue(indexExists(originalIndex)));
         assertBusy(() -> assertFalse(indexExists(secondIndex)));
         assertBusy(() -> assertEquals("true", getOnlyIndexSettings(originalIndex).get(LifecycleSettings.LIFECYCLE_INDEXING_COMPLETE)));
-    }
-
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/49073")
-    public void testRolloverAlreadyExists() throws Exception {
-        String originalIndex = index + "-000001";
-        String secondIndex = index + "-000002";
-        createIndexWithSettings(originalIndex, Settings.builder().put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
-            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0)
-            .put(RolloverAction.LIFECYCLE_ROLLOVER_ALIAS, "alias")
-        );
-
-        // create policy
-        createNewSingletonPolicy("hot", new RolloverAction(null, null, 1L));
-        // update policy on index
-        updatePolicy(originalIndex, policy);
-        // Manually create the new index
-        Request request = new Request("PUT", "/" + secondIndex);
-        request.setJsonEntity("{\n \"settings\": " + Strings.toString(Settings.builder().put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
-            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 0).build()) + "}");
-        client().performRequest(request);
-        // wait for the shards to initialize
-        ensureGreen(secondIndex);
-        // index another doc to trigger the policy
-        index(client(), originalIndex, "_id", "foo", "bar");
-        assertBusy(() -> {
-            Map<String, Object> explainIndexResponse = explainIndex(originalIndex);
-            logger.info(originalIndex + ": " + getStepKey(explainIndexResponse));
-            logger.info(secondIndex + ": " + getStepKeyForIndex(secondIndex));
-            assertThat(getStepKey(explainIndexResponse), equalTo(new StepKey("hot", RolloverAction.NAME, ErrorStep.NAME)));
-            assertThat(explainIndexResponse.get("failed_step"), equalTo(WaitForRolloverReadyStep.NAME));
-            @SuppressWarnings("unchecked")
-            String reason = ((Map<String, String>) explainIndexResponse.get("step_info")).get("reason");
-            assertThat(reason, containsString("already exists"));
-        });
     }
 
     public void testAllocateOnlyAllocation() throws Exception {

